code/game/machinery/dna_infuser/organ_sets/gondola_organs.dm

```diff
@@ -0,0 +1,138 @@
+	tongue_owner.add_mood_event("gondola_zen", /datum/mood_event/gondola_serenity)
+	. = ..()
+
+/// Loving arms: your hands become unable to hold much of anything but your hugs now infuse the subject with pax.
+/obj/item/organ/internal/liver/gondola
+	name = "mutated gondola-liver"
+	desc = "Gondola DNA infused into what was once a normal liver."
+	icon = 'icons/obj/medical/organs/infuser_organs.dmi'
+	icon_state = "liver"
+	greyscale_config = /datum/greyscale_config/mutant_organ
+	greyscale_colors = GONDOLA_COLORS
+	/// instance of the martial art granted on insertion
+	var/datum/martial_art/hugs_of_the_gondola/pax_hugs
+
+/obj/item/organ/internal/liver/gondola/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/gondola)
+	AddElement(/datum/element/noticable_organ, "left arm has small needles breaching the skin all over it.", BODY_ZONE_L_ARM)
+	AddElement(/datum/element/noticable_organ, "right arm has small needles breaching the skin all over it.", BODY_ZONE_R_ARM)
+	pax_hugs = new
+
+/obj/item/organ/internal/liver/gondola/Destroy()
+	QDEL_NULL(pax_hugs)
+	. = ..()
+
+/obj/item/organ/internal/liver/gondola/Insert(mob/living/carbon/liver_owner, special, drop_if_replaced)
+	. = ..()
+	var/has_left = liver_owner.has_left_hand(check_disabled = FALSE)
+	var/has_right = liver_owner.has_right_hand(check_disabled = FALSE)
+	if(has_left && has_right)
+		to_chat(liver_owner, span_warning("Your arms grow terribly weak as small, needle-like pricks grow all over them!"))
+	else if(has_left || has_right)
+		to_chat(liver_owner, span_warning("Your arm grows terribly weak as small, needle-like pricks grow all over it!"))
+	else
+		to_chat(liver_owner, span_warning("You feel like something would be happening to your arms right now... if you still had them."))
+	to_chat(liver_owner, span_notice("Hugging a target will pacify them, but you won't be able to carry much of anything anymore."))
+	pax_hugs.teach(liver_owner)
+	RegisterSignal(liver_owner, COMSIG_LIVING_PICKED_UP_ITEM, PROC_REF(on_owner_picked_up_item))
+	RegisterSignal(liver_owner, COMSIG_LIVING_TRY_PULL, PROC_REF(on_owner_try_pull))
+
+/obj/item/organ/internal/liver/gondola/Remove(mob/living/carbon/liver_owner, special)
+	. = ..()
+	pax_hugs.remove(liver_owner)
+	UnregisterSignal(liver_owner, list(COMSIG_LIVING_PICKED_UP_ITEM, COMSIG_LIVING_TRY_PULL))
+
+/obj/item/organ/internal/liver/gondola/proc/on_owner_picked_up_item(mob/living/carbon/owner, obj/item/picked_up)
+	SIGNAL_HANDLER
+	if(picked_up.w_class > WEIGHT_CLASS_TINY)
+		owner.balloon_alert(owner, "too weak to hold this!")
+		owner.dropItemToGround(picked_up)
+
+/obj/item/organ/internal/liver/gondola/proc/on_owner_try_pull(mob/living/carbon/owner, atom/movable/target, force)
+	SIGNAL_HANDLER
+	if(isliving(target))
+		var/mob/living/living_target = target
+		if(living_target.mob_size > MOB_SIZE_TINY)
+			owner.balloon_alert(owner, "too weak to pull this!")
```

Comment:
It's balloon alerting over the person, what is "this" referring to?

Reply:
I want to avoid directly naming `living_target` in the balloon alert, because mob names can be very, very long.

Reply:
Lord Sport Shooter Gavin XBIV moment

Reply:
I was trying to suggest the balloon alert be over what "this" is, instead of being over the owner

Reply:
oh okay sure